jsonschema/tests/test_jsonschema_test_suite.py

```diff
@@ -125,6 +128,62 @@ def leap_second(test):
     )(test)
 
 
+def ecmascript_regex_validation(test):
```

Comment:
Other drafts I think just don't run this file at all (because we don't support it), so I think it's a bit less verbose to just remove all this and not run that file for 2020 as well.

Reply:
This is true, I just saw now that `ecmascript` was ignored for all the other drafts. I removed it from the optional tests.